_i18n/es.yml

```diff
@@ -730,6 +730,7 @@ faq:
     En este sitio web hay una lista disponible de casas de cambio donde es posible
     comprar/vender Monero (XMR):'
   qavoidbc: ¿Puedo evitar descargar toda la cadena de bloques?
+  qblocksize: ¿Qué tan grande es la cadena de bloques Monero?
```

Comment:
Translations must be submitted to weblate: translate.getmonero.org

Please revert all changes to this file.